content/chainguard/chainguard-enforce/chainguard-enforce-events/index.md

```diff
@@ -1,5 +1,8 @@
 ---
-title : "Chainguard Enforce Events"
+title: "Chainguard Enforce Events"
+linktitle: "Events"
+aliases:
+- /chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-events/
```

Comment:
This file also comes from the Github workflow, so changes here will be lost. I think for this PR the easiest fix would be to leave the file in its original location, and we update the template that generates it, and the workflow separately?